### ISSUE TITLE
Explanation?

### ISSUE THREAD

**Issue by greigs** [highfidelity/hifi-blockchain#1]
I am interested in what type of blockchain implementation is to be used for the hifi currency. I don't see any code here besides some imports of the elements package and some rough notes.
Further, the last commit was almost 3 months ago.
What is the current plan for this project?

**Comment by CrackerHax** [comment]
Due to the fact that they are implementing centralized "cryptocurrency", I doubt they will open source HFC.

**Comment by greigs** [comment]
Makes sense I suppose (although disappointing). But then what is the purpose of this repository?

**Comment by CrackerHax** [comment]
Hype, probably.

**Comment by greigs** [comment]
I found this in the FAQ... https://blog.highfidelity.com/roadmap-currency-and-content-protection-8b4de8acd1f3

Q: Will my money and digital assets survive if High Fidelity goes away?
A: Yes. Because the HFC blockchain is a public ledger, anyone can (and many will) make a backup of the data. If High Fidelity were to disappear, someone else will startup a new blockchain from that backup and provide the same service of accepting new blocks to write to the ledger. Also, both High Fidelity and the blockchain software we are using are open source.

**Comment by CrackerHax** [comment]
Lol. That is a wild assumption they have made. If HiFi went away HFC would immediately become worthless from everybody dumping it like rats on a sinking ship. Anybody who knows what they are doing can make their own cryptocurrency in 5 minutes. Not sure why anybody would want to take on the task of honoring commitments of another company (e.g. they would have to download and host all the content from HiFi servers).
